Source/WebCore/svg/SVGPathParser.cpp

```diff
@@ -94,7 +94,7 @@ bool SVGPathParser::parseMoveToSegment()
 
 bool SVGPathParser::parseLineToSegment()
 {
-    auto result = m_source->parseLineToSegment();
+    auto result = m_source->parseLineToSegment(m_currentPoint);
```

Comment:
I do not understand fully the purpose of this change but I expect it is work towards other clean-up and more implementation. With this change this function becomes a little bit confusing. We are passing `m_currentPoint` to `m_source-> parseLineToSegment()`. But if `(m_pathParsingMode == NormalizedParsing` is true we are going to add `m_currentPoint` to the `result->targetPoint`. So I would expect the next step will be to make the `SVGPathSource` handle the addition of `m_currentPoint` to the result in the case of `(m_pathParsingMode == NormalizedParsing`
```
bool SVGPathParser::parseLineToSegment()
{
    auto result = m_source->parseLineToSegment(m_currentPoint);
    if (!result)
        return false;

    if (m_pathParsingMode == NormalizedParsing)
        m_consumer->lineTo(m_currentPoint, AbsoluteCoordinates);
    else
        m_consumer->lineTo(result->targetPoint, m_mode);

     m_currentPoint = result->targetPoint;
    return true;
}
```
And of course `SVGPathSource` has to be aware of `m_pathParsingMode` and `m_mode`.